addons/mail/controllers/thread.py

```diff
@@ -8,6 +8,22 @@
 
 
 class ThreadController(http.Controller):
+    @http.route("/mail/thread/access_rights", methods=["POST"], type="json", auth="user")
```

Comment:
Keep a void line above :) 
```suggestion

        @http.route("/mail/thread/access_rights", methods=["POST"], type="json", auth="user")
```